### ISSUE TITLE
E306 seems to be giving many false positives in 2.3.0

### ISSUE THREAD

**Issue by dhermes** [PyCQA/pycodestyle#621]
e.g. (a stripped down example from [`google-cloud-python`][1]) in the file `f.py`

```python
def _get_pb_likelihood(value):
    return 42.125


def f(cls, image):
    values = [image.adult, image.spoof, image.medical, image.violence]
    classifications = map(_get_pb_likelihood, values)
    return cls(*classifications)
```

which gives:

```
$ pycodestyle f.py
checking f.py
f.py:7:5: E306 expected 1 blank line before a nested definition, found 0
```

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python

**Comment by sigmavirus24** [comment]
2.3.1 was just released fixing issues with E302 and E306.

**Comment by dhermes** [comment]
Thanks a lot @sigmavirus24. Sorry I didn't notice #618 and #620

**Comment by sigmavirus24** [comment]
No worries.
